README.md

```diff
@@ -255,12 +255,15 @@ spec:
 
 There are a few variables that are customizable for awx the image management.
 
-| Name                    | Description                |
-| ----------------------- | -------------------------- |
-| tower_image             | Path of the image to pull  |
-| tower_image_pull_policy | The pull policy to adopt   |
-| tower_image_pull_secret | The pull secret to use     |
-| tower_ee_images         | A list of EEs to register  |
+| Name                      | Description                | Default                      |
+| --------------------------| -------------------------- |------------------------------|
+| tower_image               | Path of the image to pull  | quay.io/ansible/awx          |
+| tower_image_version       | Image version to pull      | 19.0.0                       |
```

Comment:
I really don't want to have to update the readme every time these variables change. I would either point to the file which contains them, or just delete the Default column altogether.